### PR TITLE
backend: Remove metrics

### PR DESCRIPTION
**Pull request by tianjing-li**
- Per Cohere internal request, removing metrics code from the open source toolkit

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -104,10 +104,6 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pyright]
-include = [
-  "src/backend/services/metrics.py", 
-  "src/backend/tools/google_drive/sync/actions/",
-  ]
 defineConstant = { DEBUG = true }
```
Comment (EugeneLightsOn) [inline pyproject.toml:107]: Do we need to check all the code? If so, I think we should set it to ./src/backend to check only the Python code.

**Review by EugeneLightsOn — COMMENTED** [review]
Looks good to me, only one small question about pyright

File: pyproject.toml (hunk 1 above)
Comment (tianjing-li) [reply pyproject.toml:110]: Let me try

Comment (tianjing-li) [reply pyproject.toml:113]: I had to revert this removal, even with the updated pyright checker it checks types across the entire backend - we have 900+ errors so there's no question that we don't have bandwidth to resolve them, I'll revert to this for now (which seems to resolve the error)

**Review by malexw — APPROVED** [review]
Nothing jumps out to me here as being a problem

**Pull request merged**